arroyo/dead_letter_queue/dead_letter_queue.py

```diff
@@ -0,0 +1,54 @@
+import logging
```

Comment:
I think this entire module should be under `arroyo/processing/strategies/`. Then you can export the symbols you need like this https://github.com/getsentry/arroyo/blob/main/arroyo/processing/strategies/streaming/__init__.py#L1-L13